crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[allow(deprecated_usage)]
```

Comment:
Can you put this on the spots where the type is actually used.

Reply:
I'm okay doing it in `coin` but definitely not in `sui_system` tests. Would that suffice?

Reply:
Done!